haas/api.py

```diff
@@ -796,20 +796,22 @@ def port_detach_nic(switch, port):
     port.nic = None
     db.session.commit()
 
-
-@rest_call('GET', '/free_nodes', Schema({}))
-def list_free_nodes():
-    """List all nodes not in any project.
+@rest_call('GET', '/node/<is_free>', Schema({'is_free' : basestring}))
```

Comment:
I think it should be /node so if for whatever reason we wanted to grab nodes by name or whatever we could do so with /node/<name>.  I'm not terribly committed to it so if you feel strongly about it and have reason, I can change it.